src/Helpers/Columnar.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace Firesphere\SolrSearch\Helpers;
+
+use SilverStripe\Dev\Debug;
+use SilverStripe\ORM\DataObject;
+use SilverStripe\ORM\FieldType\DBField;
+use SilverStripe\ORM\SS_List;
+use SilverStripe\View\ArrayData;
+
+class Columnar
```

Comment:
Where does this name come from? :O

Reply:
Can't remember, its a temporary class name :)

Reply:
For me as non native speaker it doesn't sound self explanatory

Reply:
You are correct, the code is meant to be added to Introspection class. For the meantime I added a temporary file for unit testing and still in progress.
Any name suggestion is welcome though as this class can be used not just by this module